packages/@rnw-scripts/doxysaurus/bin.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+// Yarn will fail to link workspace binaries if they haven't been built yet. Add
+// a simple JS file to forward to the CLI which is built after install.
+require('./lib-commonjs/doxysaurus');
```

Comment:
Could you update this + package.json according to this PR? https://github.com/microsoft/react-native-windows/pull/6959

I added a bit more to the JS to TS shim to also set up some global state to handle sourcemaps.

Reply:
Changed in commit https://github.com/microsoft/react-native-windows/pull/7044/commits/1e0926292e77efc7a0ba18ee1cf8d9ba8664327e